tests/utilities.py

```diff
@@ -182,7 +182,8 @@ def get_single_noise_type_config(
                     if isinstance(probability, list):
                         new_probability = [0.0 for x in probability]
                     elif isinstance(probability, dict):
-                        new_probability = {key: 0.0 for key in probability.keys()}
+                        # NOTE: this will fail default config validations
```

Comment:
Can you explain this? Why would the key be an integer? I don't really undwerstand your note, either.

Reply:
This structure is for "possible age differences" where the keys can be -2 or 1 to indicate what int to add to a simulant's actual age and the value is the probability of picking each age difference.